open_spiel/python/games/domino.py

```diff
@@ -0,0 +1,346 @@
+import copy
+import itertools
+
+import numpy as np
+
+import pyspiel
+
+_NUM_PLAYERS = 2
+
+# The first player to play is the one holding the highest rank tile.
+# The rank of tiles is the following:
+#   1. Highest double.
+#   2. If none of the players hold a double, then highest weight.
+#   3. If the highest weighted tile of both players has the same weight
+#      then the highest single edge of the highest weighted tile.
+
```

Comment:
Can you also add the game to the games list? In docs/games.md.

Reply:
Please fix